Studies/python/BayesianOptimizationCustom.py

```diff
@@ -0,0 +1,179 @@
+# Definition of the model optimization using Bayesian optimization as a method
+# This file is part of https://github.com/hh-italian-group/hh-bbtautau.
+
+from bayes_opt import UtilityFunction
+from bayes_opt import BayesianOptimization
+from bayes_opt.observer import JSONLogger
+from bayes_opt.event import Events
+
+import json
+import copy
+import numpy as np
+
+class BayesianOptimizationCustom:
+    def __init__(self, param_ranges_json, params_init_probe_json, fn, probed_points_target_output,
+                 probed_points_opt_output, num_iter, random_state, load_points, probed_points_target_output_prev='',
+                 probed_points_opt_output_prev=''):
```

Comment:
remove `probed_points_target_output_prev` and `probed_points_opt_output_prev` from the constructor